### ISSUE TITLE
lesson page: introductive elements of text

### ISSUE THREAD

**Issue by spapastamkou** [C2DH/ranketwo#228]
As per #223, this issue will serve to reflect on and reorganize the introductive text elements of the lessons individual page.

**Comment by spapastamkou** [comment]
Currently, the text elements that can be considered as introductive to a lesson are the following: 
- a short text under the title of the lesson (and currently preceded by the names of the curators. Location: file `/_unit/LESSON-SLUG.md` This text can be considered as a very short abstract and sometimes it can repeat a text present in the introduction or the objective part that follows. 
- introduction: it is essentially an abstract - can be more or less lengthy from lesson to lesson.
- lesson objective: can range from one phrase to almost a sort of abstract from lesson to lesson
- content organization: explains (currently) the S - M- L structure of the lessons that is now obsolete; sometimes offers descriptions/objectives for animation and series of assignments. The latter information can be integrated in a lesson abstract (see proposal below). The content organization part is the object of #223 (but I may have to reorganize the work after this issue).   

![social-media-01](https://user-images.githubusercontent.com/27706167/231806685-071629de-3220-47e2-ba91-34365420d5f0.png)
I attach a screenshot of the page of the lesson on social media that has all the text elements.

**Comment by spapastamkou** [comment]
Here is a list of what I consider as problems with this organization: 
- the text under the title of the lesson can be redundant with what follows: sometimes it is formulated exactly in the same way (repetition) or can formulate in different terms the same thing (lack of coherence). 
- introduction: globally the texts are short (1-3 phrases), which is good! Sometimes the intro repeats the text under the title: sometimes it is absent (see archival to digital). 
- lesson objective:

**Comment by spapastamkou** [comment]
A draft proposal on how to reorganize this part of the page. 
In any case, the "how the content is organized" part should be removed and it is the object of another issue here (referenced above). 
The idea is to have only one part that could be under the title "Introduction" (for practical reasons, to not change the names of the files) as now, but no longer have separate parts for objectives and organization of the content. Instead, the Introduction part should follow a clearly defined pattern: 
- state the general purpose of the lesson: what type of sources does it concern, which part of a historical research (heuristics, hermeneutics) if possible, what relevance with digital history...
- make explicit the learning objectives by staying very concrete about them - a user should be able to understand what they gain in skills/knowledge after completing a lesson
- a brief announcement of the structure of the lesson
- provide bibliographical references in the intro should be avoided; if however the author(s) wishes to recommend preliminary readings, this should be stated in a phrase and a hyperlink could send to the bibliography part of the lesson, where the introductory readings should be registered separately (to have an anchor link). 

A very short and simple abstract of one phrase, maximum two, should be kept under the title of the lesson. 

Remove the button: go to assignments
Integrate a table of contents (toc) below to go to: intro, animation, assignments (or main part of the lesson), bibliography if any. 

On the right of the lesson page, where currently is shown a toc, make visible the contributors (or with another configuration, but metadata on contributors should be visible on the lesson page, making use of structured metadata, rather than in the interior as now.

**Comment by spapastamkou** [comment]
This issue has been discussed with @danieleguido on 18 April. As for how we are to reorganize this part of the lesson page (see [draft proposal above](https://github.com/C2DH/ranketwo/issues/228#issuecomment-1512607212)). We agreed to most of it but our idea is to finally keep the toc on the right of the page, and make the contributors visible under the title and the short abstract of the lesson. We did not discuss the button" go to lessons". More details are found in the minutes of the meeting.

**Comment by spapastamkou** [comment]
As a result of this issue having emanated from #223, the edits that will close it pass through a branch named after the previous issue: [`issue-223-update-intro-files`](https://github.com/C2DH/ranketwo/tree/issue-223-update-intro-files) I note this here to avoid confusion.

**Comment by spapastamkou** [comment]
This has been fixed in pull request #227. I modified the introduction files of all the lessons. The introductions of in German of the two lessons that currently exist in this language should be verified by a native-German speaker - I will open a separate issue for this. 
Otherwise, for the lessons that are not yet translated, I copied the updated texts in the files so their translations should take into account directly the translated text (this was not the case of the lesson of historical newspapers that was translated before the into text was updated). As it is not yet published, I will deal with the updates of the intros in DE and FR in the moment of the creation of the respective files. 
Finally, this issue mentions also the possibility to change the aspect of the tocs mainly to be able to see the content of the assignments. We leave this as it is for the moment (to avoid having tocs that are much too long) but if the possibility exists to deploy a more detailed sub-toc once we click on the assignments, then follow the link for each of them, we should explore this with @danieleguido.
